packages/kit/src/runtime/server/page/render.js

```diff
@@ -292,10 +296,11 @@ export async function render_response({
 
 		const blocks = [];
 
+		// TODO only populate `env` if `$env/dynamic/public` is used in the app
```

Comment:
Is this TODO from an earlier iteration of the PR? The below code looks like it only populates env if it's used

Reply:
Yep, this is our of date